src/Process/Builder/InitialTestRunProcessBuilder.php

```diff
@@ -77,9 +73,7 @@ public function createProcess(
 
         $process->setTimeout(null); // ignore the default timeout of 60 seconds
 
-        $symfonyProcessVersion = $this->versionParser->parse(Versions::getVersion('symfony/process'));
-
-        if (version_compare($symfonyProcessVersion, '4.4.0', '<')) {
+        if (method_exists($process, 'inheritEnvironmentVariables')) {
```

Comment:
This function should be imported (`use function method_exists;`)

Reply:
fixed